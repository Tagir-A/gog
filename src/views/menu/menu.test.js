import React from "react";
import { shallow, mount, render } from "enzyme";
import Menu from "./menu";

it("renders without crashing", () => {
  const wrapper = shallow(<Menu />);
  expect(wrapper.debug()).toMatchSnapshot();
});
