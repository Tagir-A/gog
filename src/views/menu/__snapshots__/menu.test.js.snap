// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
"<header className=\\"menu\\">
  <div className=\\"menu__container\\">
    <a className=\\"menu__logo\\" href=\\"https://www.gog.com/\\">
      <img src=\\"logo-classic.svg\\" alt=\\"main page link\\" />
    </a>
    <Connect(Cart) />
  </div>
</header>"
`;
